src/base/bittorrent/session.cpp

```diff
@@ -5158,7 +5166,8 @@ void Session::handleSessionStatsAlert(const lt::session_stats_alert *p)
     const auto calcRate = [interval](const qint64 previous, const qint64 current) -> qint64
     {
         Q_ASSERT(current >= previous);
-        return ((current - previous) / interval);
+        Q_ASSERT(interval >= 0);
+        return (((current - previous) * 1000000) / interval);
```

Comment:
I'm concerned that this might overflow (* 1000000), just keep an eye out for user report about it.

Reply:
> I'm concerned that this might overflow (* 1000000), just keep an eye out for user report about it.

Ok.
I could extract division to be performed before multiplication but it can reduce accuracy so I'll do it if there are feedbacks from users.

Reply:
>I could extract division to be performed before multiplication but it can reduce accuracy 

It will be too inaccurate to be useful so that probably won't work. We can deal with it later when it really happens.